src/rev.md

```diff
@@ -19,58 +19,63 @@ Both the inplace and "normal" variant return the gradient. The difference is tha
 ### Usage story
 Let us start by looking at the most basic examples we can think of:
 
-$f(x,y) = x^2 + y*3$
+\\[ f(x,y) = x^2 + 3y \\]
 
-We have two input variables $x$, $y$ and a scalar return value.  Just to check our sanity, we first pass it to [wolfram alpha](https://www.wolframalpha.com/input?i2d=true&i=D%5BPower%5Bx%2C2%5D+%2B+y*3%2Cx%5D%3B+D%5BPower%5Bx%2C2%5D+%2B+y*3%2Cy%5D%3B+). No big surprises so far. Let's check for Enzyme (our compiler explorer does not handle Rust yet, so you'l have to trust me on this).
+We have two input variables \\(x\\), \\(y\\) and a scalar return value.  Just to check our sanity, we first pass it to [wolfram alpha](https://www.wolframalpha.com/input?i2d=true&i=D%5BPower%5Bx%2C2%5D+%2B+y*3%2Cx%5D%3B+D%5BPower%5Bx%2C2%5D+%2B+y*3%2Cy%5D%3B+). No big surprises so far. Let's check for Enzyme (our compiler explorer does not handle Rust yet, so you'l have to trust me on this).
```

Comment:
mdBook doesn't support `$x$` notation. https://rust-lang.github.io/mdBook/format/mathjax.html

Reply:
nice, I didn't scroll down that far and was already looking for plugins yesterday